src/overlays/actors/ovl_En_Az/z_en_az.c

```diff
@@ -5,28 +5,69 @@
  */
 
 #include "z_en_az.h"
+#include "objects/object_az/object_az.h"
+#include "overlays/actors/ovl_En_Twig/z_en_twig.h"
+#include "overlays/actors/ovl_En_Fish/z_en_fish.h"
 
 #define FLAGS (ACTOR_FLAG_10 | ACTOR_FLAG_80000000)
 
 #define THIS ((EnAz*)thisx)
 
+typedef struct {
+    /* 0x0 */ s16 unk_0;
+    /* 0x4 */ f32 unk_4;
+} struct_80A98F94; // size = 0x8
+
 void EnAz_Init(Actor* thisx, PlayState* play);
 void EnAz_Destroy(Actor* thisx, PlayState* play);
 void EnAz_Update(Actor* thisx, PlayState* play);
 void EnAz_Draw(Actor* thisx, PlayState* play);
 
+void func_80A982E0(PlayState* play, ActorPathing* actorPathing);
+void func_80A98414(EnAz* this, PlayState* play);
+s32 func_80A98DA4(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx);
+void func_80A98E48(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx);
+void func_80A98EFC(EnAz* this, PlayState* play, u16 textId, s32 arg3, s32 arg4);
+void func_80A98F94(struct_80A98F94* yData, f32 frame, f32* yInterp);
+
+void func_80A95C5C(EnAz* this, PlayState* play);
 void func_80A95CEC(EnAz* this, PlayState* play);
+
+void func_80A95DA0(EnAz* this, PlayState* play);
 void func_80A95E88(EnAz* this, PlayState* play);
+
+void func_80A95F94(EnAz* this, PlayState* play);
 void func_80A95FE8(EnAz* this, PlayState* play);
+
+void func_80A979DC(EnAz* this, PlayState* play);
 void func_80A979F4(EnAz* this, PlayState* play);
+
+void func_80A97A28(EnAz* this, PlayState* play);
 void func_80A97A40(EnAz* this, PlayState* play);
+
+void func_80A97A9C(EnAz* this, PlayState* play);
 void func_80A97AB4(EnAz* this, PlayState* play);
+
+void func_80A97C0C(EnAz* this, PlayState* play);
 void func_80A97C24(EnAz* this, PlayState* play);
 void func_80A97C4C(EnAz* this, PlayState* play);
+
+void func_80A97D5C(EnAz* this, PlayState* play);
 void func_80A97E48(EnAz* this, PlayState* play);
+
+void func_80A97EAC(EnAz* this, PlayState* play);
 void func_80A97F9C(EnAz* this, PlayState* play);
 
-#if 0
+static AnimationSpeedInfo D_80A99010[14] = {
```

Comment:
```suggestion
static AnimationSpeedInfo sAnimationInfo[] = {
```
There's an open PR making these consistent, might as well sync with that